onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -286,56 +290,38 @@ def time_fn(args, fn, inputs):
         outputs = fn(inputs)
         logger.info(outputs)
 
-    input_sync = (  # noqa: E731
-        lambda *kwargs: args.io_binding.synchronize_inputs()
-        if args.device != "cpu" and args.benchmark_type in {"ort-msft", "ort-convert-to-onnx"}  # ORT synchronize
```

Comment:
We need to revert these changes for `input_sync` and `output_sync` as they are needed for other EPs.

Reply:
will do, these seems be removed by accident, thanks for the catch